README.md

```diff
@@ -0,0 +1,96 @@
+# HTML & CSS3 Course
```

Comment:
Your READ.me file is not professional. Please go through this [link](https://github.com/microverseinc/readme-template) for the template.